pkgs/by-name/un/unimatrix/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, python3
+, fetchFromGitHub
+, git
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
+    # Needed for setuptools-scm to detect version
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools-scm
+    python3.pkgs.pip
+    git
+  ];
+
+  meta = with lib; {
+    description = "Python script to simulate the display from \"The Matrix\" in terminal.";
+    homepage = "https://github.com/will8211/unimatrix";
+    license = licenses.gpl3;
```

`licenses.gpl3` is deprecated in favor of `licenses.gpl3Only` and `licenses.gpl3Plus`. You can read more about it here: https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206

```suggestion
    license = licenses.gpl3Only;
```

---

Good to know. Done.